ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -658,15 +668,16 @@ double UncoupledProcessesTimeLoop::computeTimeStepping(
         }
         else
         {
-            if (std::abs(dt -prev_dt) < std::numeric_limits<double>::min()
-                && _last_step_rejected)
+            if (std::abs(dt - prev_dt) < std::numeric_limits<double>::min() &&
+                _last_step_rejected)
             {
-                OGS_FATAL("\tThis time step is rejected and the new computed"
-                          " step size is the same as\n"
-                          "\tthat was just used.\n"
-                          "\tSuggest to adjust the parameters of the time"
-                          " stepper or try other time stepper.\n"
-                          "\tThe program stops");
+                OGS_FATAL(
+                    "\tThis time step is rejected and the new computed"
+                    " step size is the same as\n"
+                    "\tthat was just used.\n"
+                    "\tSuggest to adjust the parameters of the time"
+                    " stepper or try other time stepper.\n"
+                    "\tThe program stops");
```

Comment:
Good point.

This piece of source code have moved from this function to loop(...), and it is documented now (see  commit  2191287).